server/lostandfound/urls.py

```diff
@@ -18,5 +18,5 @@
 
 admin.site.site_header = "Lost and Found Administration"
 urlpatterns = [
-    url(r'^admin/', admin.site.urls),
+    url(r'^admin', admin.site.urls),
```

Comment:
for `localhost:8000/admin/`, the page wont be found based on the new changes made on the route. User tend to input this url `localhost:8000/admin` more than its counterpart, but we can still take care of the edge case you highlighted by including a second route to take care of the odds.